README.md

```diff
@@ -98,7 +99,7 @@ CIP Editors meetings are public, recorded, and [published on Youtube](https://ww
 | 1855 | [Forging policy keys for HD Wallets](./CIP-1855/) | Active |
 | 9999 | [Cardano Problem Statements](./CIP-9999/) | Active |
 
-<p align="right"><i>Last updated on 2023-09-08</i></p>
+<p align="right"><i>Last updated on 2023-09-19</i></p>
```

Comment:
```suggestion
```
Do we even need this date? I always thought its not needed considering the lower date

Reply:
I don't think any of these dates for lists of PRs are needed but only assuming a general solution... within easy reach I think... of generating the lists dynamically through tagging according to their precise editorial status.

I have a planned improvement as part of my Catalyst proposal to introduce such a tagging vocabulary & until such a time comes I would happily see those "last updated" dates dropped since the only one that probably matters to the general public is the list of merged CIPs (which generally aren't obtainable from PRs).

But until then I think we need to continue the tradition of putting 1 "last update" date at the footer of each 1 table.  The alternative to this (I think evidenced before @Ryun1 you arrived) was too difficult to interpret & wasn't modified consistently.